### ISSUE TITLE
create nice project raport

### ISSUE THREAD

**Issue by StickStickens** [StickStickens/segmentation-inpainting#10]
Requirements:
- description of the data set, with a few image examples
- description of the problem
- description of used architectures with diagram showing the layers; For large models containing blocks, the blocks and the connections between them can be shown separately.
- model analysis: size in memory, number of parameters,  
- description of the training and the required commands to run it
- description of used metrics, loss, and evaluation
- plots: training and validation loss, metrics
- used hyperparameters along with an explanation of each why such value was chosen
- comparison of models
- list of libraries and tools used can be a requirements.txt file
- a description of the runtime environment
- training and inference time,
- preparation of a bibliography - the bibliography should contain references to the data set (preferably the article in which the collection was presented) and all scientific works and studies, including websites with tips on the solution.
- A table containing all the completed items with points.
- Link to Git
